test/unit/test_float.c

```diff
@@ -1414,7 +1472,86 @@ bool f32_ieee_cast_test(void) {
 	mu_end;
 }
 
-bool f80_round_test(void) {
+bool f80_ieee_add_test(void) {
+#if (__i386__ || __x86_64__) && !__WINDOWS__
+	RzFloat *x_f80 = rz_float_new_from_f80(2.59958250082224806804l);
```

Comment:
Could those tests be written with `rz_float_new_from_bv()` instead of `from_f80`? Then they would be portable. `rz_float_new_from_f80()` could be (or is already perhaps) tested individually.

Reply:
I see, that makes sense. I'll do this.